tools/mobrpg/mobrpg/commands/pull.py

```
@@ -0,0 +1,205 @@
+# visible to a "what's new" report. landfeature/item have NO `/type` endpoint —
+# their types live on the elements — so they are excluded here.
+TYPE_KINDS = ["creature/type", "organization/type", "political/type"]
+
+
+def _list_all(world: str, kind: str, token: str) -> list:
+    try:
+        r = client._request("GET", f"/world/{world}/{kind}", token=token,
+                            query={"page": 0, "size": 500})
+        c = r.get("content", r) if isinstance(r, dict) else r
+        return c if isinstance(c, list) else []
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)